streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignorConfiguration.java

```diff
@@ -267,25 +267,46 @@ public static class AssignmentConfigs {
         public final int numStandbyReplicas;
         public final long probingRebalanceIntervalMs;
         public final List<String> rackAwareAssignmentTags;
+        public final Integer rackAwareAssignmentTrafficCost;
```

Comment:
It can be null and we will use default in assignor. It's related to your questions in https://github.com/apache/kafka/pull/14139#discussion_r1284910181